### ISSUE TITLE
Module compilation error

### ISSUE THREAD

**Issue by mhjiang97** [nf-core/spatialvi#22]
### Description of the bug

A bug appears when I tested the pipeline.

### Command used and terminal output

```console
Command:

nextflow run nf-core/spatialtranscriptomics -profile test,docker -r dev
```

Output:
```
N E X T F L O W  ~  version 22.04.5
Launching `https://github.com/nf-core/spatialtranscriptomics` [amazing_wescoff] DSL2 - revision: 40e2fe8125 [dev]


------------------------------------------------------
                                        ,--./,-.
        ___     __   __   __   ___     /,-._.--~'
  |\ | |__  __ /  ` /  \ |__) |__         }  {
  | \| |       \__, \__/ |  \ |___     \`-._,-`-,
                                        `._,._,'
  nf-core/spatialtranscriptomics v1.0dev
------------------------------------------------------
Core Nextflow options
  revision                  : dev
  runName                   : amazing_wescoff
  containerEngine           : docker
  launchDir                 : /home/jmh/tmp
  workDir                   : /home/jmh/tmp/work
  projectDir                : /home/jmh/.nextflow/assets/nf-core/spatialtranscriptomics
  userName                  : jmh
  profile                   : test,docker
  configFiles               : /home/jmh/.nextflow/assets/nf-core/spatialtranscriptomics/nextflow.config

Input/output options
  input                     : https://raw.githubusercontent.com/nf-core/test-datasets/spatialtranscriptomics/testdata/test-dataset-subsampled/samplesheet.csv

Institutional config options
  config_profile_name       : Test profile
  config_profile_description: Minimal test dataset to check pipeline function

Max job request options
  max_cpus                  : 4
  max_memory                : 6.GB
  max_time                  : 1.h

!! Only displaying parameters that differ from the pipeline defaults !!
------------------------------------------------------
If you use nf-core/spatialtranscriptomics for your analysis please cite:

* The nf-core framework
  https://doi.org/10.1038/s41587-020-0439-x

* Software dependencies
  https://github.com/nf-core/spatialtranscriptomics/blob/master/CITATIONS.md
------------------------------------------------------
Project directory:  /home/jmh/.nextflow/assets/nf-core/spatialtranscriptomics

Module compilation error
- file : /home/jmh/.nextflow/assets/nf-core/spatialtranscriptomics/./workflows/../subworkflows/local/../../modules/local/tasks.nf
- cause: unable to resolve class JsonSlurper
 @ line 201, column 19.
       sample_info = new JsonSlurper().parse(new File(fileName))
                     ^

Module compilation error
- file : /home/jmh/.nextflow/assets/nf-core/spatialtranscriptomics/./workflows/../subworkflows/local/../../modules/local/tasks.nf
- cause: unable to resolve class JsonSlurper
 @ line 339, column 19.
       sample_info = new JsonSlurper().parse(new File(fileName))
                     ^

2 errors
```
```


### Relevant files

_No response_

### System information

- Nextflow version: 22.04.5
- Container engine: Docker
- OS: CentOS Linux
- Version of nf-core/spatialtranscriptomics: dev

**Comment by mhjiang97** [comment]
what columns `check_samplesheet.py` will check don't seem to match the README.

```
HEADER = ["sample","tissue_positions_list","tissue_lowres_image","tissue_hires_image","scale_factors","barcodes","features","matrix"]
```

**Comment by fasterius** [comment]
Thanks for the bug report! This is fixed in #17.
